Sources/TensorFlow/Operators/Math.swift

```diff
@@ -2181,3 +2181,34 @@ internal extension Tensor where Scalar: TensorFlowFloatingPoint {
         _vjpMatmul(lhs, rhs)
     }
 }
+
+public extension Tensor where Scalar: TensorFlowFloatingPoint {
+    /// Returns the QR decomposition of each inner matrix in the tensor, a tensor with inner 
+    /// orthogonal matrices q and a tensor with inner upper triangular matrices r, such that the 
+    /// tensor is equal to matmul(q, r).
+    /// 
+    /// - Parameters:
+    ///   - fullMatrices: If true, compute full-sized `q` and `r`. If false
+    ///     (the default), compute only the leading `P` columns of `q`.
```

Comment:
- Identifier `P` is undefined.
- Could you help make sure this documentation item fits to size (100 columns)?
```suggestion
    ///     (the default), compute only the leading `min(shape[rank - 1], shape[rank - 2])` columns of `q`.
```